.github/workflows/run_tests.yml

```diff
@@ -25,10 +25,8 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install packaging
-          python setup.py develop
-          pip install '.[test,plots]'
-          pip install tensorflow xgboost codecov torch torchvision pylint sentencepiece
+          pip install -e '.[test,plots]'
+          pip install tensorflow
```

Comment:
I'll address this in a separate PR. Moving this into the `test` optional dependencies to resolve #30.